extension/js/content_scripts/webmail/gmail-element-replacer.ts

```diff
@@ -206,6 +205,29 @@ export class GmailElementReplacer implements WebmailElementReplacer {
     }
   };
 
+  private parseBlocksFromEmailContainer = (emailContainer: HTMLElement) => {
+    const blocksFromEmailContainer = Mime.processBody({ text: emailContainer.innerText });
```

Comment:
We may introduce below inner function to avoid `Mime.processBody({...}`
```ts
const parseText = (text: string) => Mime.processBody({ text });
```

And then write like below
```ts
const blocksFromEmailContainer = parseText(emailContainer.innerText);
```

Reply:
Nice 👍